README.md

```diff
@@ -136,10 +136,18 @@ If you have installed Anaconda, these should already be available to you.
 
     git clone https://github.com/fonnesbeck/ngcm_pandas_2016.git
 
-Make sure you have the requirements installed.
+If you are not familiar with Git and GitHub, you can simply download the zip file of the repository at the top of the main repository page.
 
-    cd ngcm_pandas_course
+Then, move to the directory created by the clone/zip file:
 
-If using pip,
+    cd ngcm_pandas_2016
 
-    pip install -r requirements.txt
+and install everything using `conda`:
+
+    conda env create -f environment.yml
+    
```

Comment:
Done. I am going to merge this so that the updated instructions are available. Feel free to modify further.